pkgs/development/cuda-modules/cuda-library-samples/extension.nix

```diff
@@ -1,5 +1,12 @@
-{ hostPlatform, lib }:
+{
+  lib,
+  # NOTE: We would use backendStdenv to get the hostPlatform, but it's not available in the callPackage context
+  # we're using to call this file.
```

Comment:
I brought this up before but lost the thread: this comment doesn't sound right; this is called from `cudaPackages.callPackage` and surely `hasAttr "backendStdenv" cudaPackages`